dev/breeze/src/airflow_breeze/commands/setup_commands.py

```diff
@@ -260,7 +260,11 @@ def dict_hash(dictionary: dict[str, Any]) -> str:
     """MD5 hash of a dictionary. Sorted and dumped via json to account for random sequence)"""
     # noinspection InsecureHash
     dhash = hashlib.md5()
-    encoded = json.dumps(dictionary, sort_keys=True, default=vars).encode()
+    try:
+        encoded = json.dumps(dictionary, sort_keys=True, default=vars).encode()
+    except TypeError:
+        get_console().print(dictionary)
```

Comment:
```suggestion
        get_console().print(repr(dictionary))
```

Since this is TypeError (not ValueError) a repr print would probably be a bit more useful? I’m just guessing here.

Reply:
I think not needed, this is `rich` print which already has all kinds of magic to nicely print whatever we pass to it, so wrapping it with repr will loose the information that otherwise `rich` library could use to give us nicer and more user friendly representation.